### ISSUE TITLE
a question about Equation 5

### ISSUE THREAD

**Issue by wangjs9** [yingtaomj/Reasoning-in-Dialog#4]
Hi~ Have read this wonderful paper, I have a question about equation 5. In equation 5, why the k is from 1 to N_j. I think that k should be from 1 to N-q.

**Comment by iriscxy** [comment]
Yes, you are right, k is from 1 to N_q. Thanks for pointing it out!
